tests/test_pdf_qgauss.py

```diff
@@ -0,0 +1,60 @@
+#  Copyright (c) 2024 zfit
+import numpy as np
+import pytest
+import zfit
+from zfit import z
+import zfit.z.numpy as znp
+from numba_stats import qgaussian as qgaussian_numba
+import tensorflow as tf
+
+mu_true = 0.5
+sigma_true = 1.1
+true_integral_dict = {1.00001: 0.9999782, 1.5: 0.9695, 2.0: 0.8063152, 2.5: 0.4805167, 2.9: 0.1078687, 2.99: 0.011044}
+
+
+def create_qgauss(q, mu, sigma, limits):
+    obs = zfit.Space("obs1", limits)
+    qgauss = zfit.pdf.QGauss(q=q, mu=mu, sigma=sigma, obs=obs)
+    return qgauss, obs
+
+
+@pytest.mark.parametrize("q", [1.00001, 1.5, 2.0, 2.5, 2.9, 2.99])
+def test_qgauss_pdf(q):
+    qgauss, _ = create_qgauss(q=q, mu=mu_true, sigma=sigma_true, limits=(-5, 5))
+    assert qgauss.pdf(0.5, norm=False).numpy().item() == pytest.approx(
+        qgaussian_numba.pdf(0.5, q=q, mu=mu_true, sigma=sigma_true), rel=1e-5
+    )
+    np.testing.assert_allclose(
+        qgauss.pdf(tf.range(-5, 5, 10_000), norm=False).numpy(),
```

Comment:
maybe just factor out the `tf.range` calls as `testdata` or similar? just reduces the number of calls ( not a big game changer oc)